generator/src/main/java/com/scottlogic/deg/generator/walker/RuntimeDecisionTreeWalkerFactory.java

```diff
@@ -39,12 +42,23 @@ public DecisionTreeWalker getDecisionTreeWalker(Path outputPath) {
                     rowSpecMerger,
                     new ExhaustiveProducer());
             case REDUCTIVE:
```

Comment:
Can we introduce a new mode which is different to reductive for now. Then we have the ability to review REDUCTIVE in isolation from COMBINATION.